doc/classes/@GlobalScope.xml

```diff
@@ -2020,16 +2020,16 @@
 			Help key.
 		</constant>
 		<constant name="KEY_BACK" value="4194376" enum="Key">
-			Media back key. Not to be confused with the Back button on an Android device.
+			Back key.
```

Comment:
Are you sure about this? This was documented by @Calinou in #41581 based on this comment: https://github.com/godotengine/godot/issues/19325#issuecomment-394155128

Did the mapping change since then?

Reply:
At least on Linux it seems to refer to the "Internet" browsing keys of the keyboards of yore: https://gitlab.freedesktop.org/xorg/proto/xorgproto/-/blob/master/include/X11/XF86keysym.h#L64

![image](https://github.com/user-attachments/assets/dc19f445-cdd0-4b07-bfad-8f9968a83265)

As well as possibly the (also history-related) "back" key on some Chromebooks keyboards, according to our code (`scancode_map[0xA6] = Key::BACK; // On Chromebooks`).

![image](https://github.com/user-attachments/assets/35045228-c7c7-40be-9b20-b93f3e890e86)

Seems to be a similar situation on Windows: `vk_map[VK_BROWSER_BACK] = Key::BACK; // (0xA6)`

And on Android it's indeed
```
47:     { AKEYCODE_BACK, Key::BACK }, // (4) Back key.
```
But the Android NDK docs don't really say more about this. They also have a mapping for `AKEYCODE_FORWARD`, what's a Forward key on an Android device?

We would need to check whether #19325 is still true or not.

Reply:
> Android mouse's `BUTTON_FORWARD` and `BUTTON_BACK` buttons are dispatched as key events.

I may be wrong but to me it looks like this PR makes it so that those Android-exclusive inputs are sent through `KEY_BACK` and `KEY_FORWARD` inputs.

With that said, please do not change the description to be unusually succinct again. On desktop platforms these keys _really_ refer to the Media keys, which are definitely not the same as the additional mouse buttons.
At least clarify that on Android, these correspond to certain other keys.

Reply:
Oh boy, after reading your reply, there's a whole rabbit hole to dissect about the purpose of these keys on various platforms, huh.

Reply:
> They also have a mapping for AKEYCODE_FORWARD, what's a Forward key on an Android device?

The forward key on Android is mapped to [`KEYCODE_FORWARD`](https://developer.android.com/reference/android/view/KeyEvent#KEYCODE_FORWARD): 

>> Key code constant: Forward key. Navigates forward in the history stack. Complement of [KEYCODE_BACK](https://developer.android.com/reference/android/view/KeyEvent#KEYCODE_BACK).

> We would need to check whether https://github.com/godotengine/godot/issues/19325 is still true or not.

I don't think the documentation update was ever correct on Android. `back` events on Android are special but are still sent as `back` key events and dispatched via numerous devices (e.g: touch button, mouse, keyboard, etc..), so the mapping in Godot should indeed be `KEY_BACK` and should be available to the game logic which this PR fixes.